src/Microsoft.AspNetCore.Mvc.RazorPages/RazorPagesOptions.cs

```diff
@@ -42,5 +43,31 @@ public string RootDirectory
                 _root = value;
             }
         }
+
+        public bool DisableAreas { get; set; }
```

Comment:
@Eilon \ @mkArtakMSFT naming suggestions

Reply:
@pranavkm, RespectAreas or IgnoreAreas?

Reply:
Spoke to @DamianEdwards and he suggested using `EnableAreas`.